src/index.html

```diff
@@ -3,11 +3,47 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="images//imac.jpeg"
+        alt="imac"
+        class="card__items card__items--logo"
+      >
```

Comment:
[Style rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules):
- alt attribute should describe the image content
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/23c955e9-44d3-43db-91de-ccc1cb12e460)
- You don't need modifier in this case, because it's only one variant of this image, just `card__logo` or something similar